nccl/tests/nccl-single-node.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+#
+# Copyright 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+#
+
+set -e
+
+echo "'INFO' ==> Starting preparation for NCCL single-node test"
+
+source $WORKSPACE/libfabric-ci-scripts/nccl/common/nccl-common.sh
+
+# Number of nodes used for nccl tests
+NUM_NODES=1
+
+# Each node has 8 GPUS
+NUM_GPUS=$(( ${NUM_NODES} * 8 ))
+
+set_jenkins_variables
+
+trap 'on_exit'  EXIT
+
+for region in ${aws_regions[@]}; do
+
+    # Set the default region
+    set_aws_defaults ${region}
+
+    custom_instance_preparation
+
+    # Create instance for custom AMI preparation
+    echo "==> Launching instance in region ${AWS_DEFAULT_REGION}"
+    create_instance ${SSHSG} 1 ${prep_ami} ${instance_ami_type}
+
+    # In case of low capacity switch to another region
+    if [ ${create_instance_exit_code} -ne 0 ]; then
+        delete_sg ${SSHSG}
+        delete_sg ${SGId}
+        echo "==> Changing the region"
+        continue  
+    else
+        break 
+    fi
+done
+
+test_instance_status ${INSTANCE_IDS}
+
+test_ssh ${INSTANCE_IDS}
+
+# Install software and prepare custom AMI
+prepare_ami "${PULL_REQUEST_REF}" "${PULL_REQUEST_ID}" "${TARGET_BRANCH}" "${TARGET_REPO}" "${PROVIDER}"
+
+# Upload AMI to marketplace
+create_ami ${INSTANCE_IDS}
+
+# Terminate instance used for AMI preparation
+terminate_instances
+
+# We need placement group for NCCL testing
+ENABLE_PLACEMENT_GROUP=1
```

Comment:
General nit for the PR: define static variable definitions up front in the file so it is easy to find them all.

Reply:
In this particular case we need to enable this parameter on demand.